proposed/2023/Add-NoHttpCacheOption.md

```diff
@@ -0,0 +1,61 @@
+# Title
+
+- Author Name (https://github.com/Nigusu-Allehu)
+- Start Date (YYYY-MM-DD)
```

Comment:
I'd just remove this. I'll remove it from the template. Not sure there's any benefit to it.

I created a PR to improve this: https://github.com/NuGet/Home/pull/12853